backend/streetdrop-api/src/main/java/com/depromeet/domains/recommend/service/SearchRecommendService.java

```diff
@@ -0,0 +1,24 @@
+package com.depromeet.domains.recommend.service;
+
+import com.depromeet.domains.recommend.dto.response.SearchTermRecommendResponseDto;
+import com.depromeet.domains.recommend.repository.SearchRecommendTermRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+@Service
+@RequiredArgsConstructor
+public class SearchRecommendService {
+
+    private final SearchRecommendTermRepository searchRecommendTermRepository;
+
+    public SearchTermRecommendResponseDto recommendSearchTerm() {
+        var recommendTerm = searchRecommendTermRepository.getRandomSearchRecommendTerm().orElseThrow(
+                () -> new RuntimeException("검색 추천어가 존재하지 않습니다.")
+        );
+
+        var description = recommendTerm.getDescription();
+        var termList = recommendTerm.getTerms();
```

Comment:
```suggestion
        var terms = recommendTerm.getTerms();
```
현재 상황에서, 자료구조 타입을 통해 변수를 지정하는 것보다는 복수형 네이밍을 가져가는게 더 적합할 것 같다고 느껴지네요.

Reply:
갑사합니다